tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -231,6 +262,26 @@ install_kernel_tdx_experimental() {
 		"-x tdx -u ${kernel_url}"
 }
 
+#Install sev kernel asset
+install_kernel_sev() {
```

Comment:
similar question as initrd

kernel function template to reduce future code duplication?

might depend on which tee goes first

Reply:
Agree,  on CCv0 there is the `install_cc_tee_kernel()` function to handle whatever TEE.

Reply:
I also vouch for this one, but it can be done in a follow-up PR.